### PR TITLE
Correction2

### PR DESCRIPTION
**Pull request by bikashstackmybiz**
Erercises on 24

### CODE REVIEW

File: Exception2.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+public class Exception2 {
+    
+        public static void main (String[] args) {
+            int a=10;
```
Comment (rakesh-stackmybiz) [inline Exception2.java:4]:
```suggestion
            int a = 10;
```

File: Exception2.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+            for(int i=5;i>=0;i--)
```
Comment (rakesh-stackmybiz) [inline Exception2.java:5]:
```suggestion
            for(int i=5; i>=0; i--)
```

File: Exception2.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+               try{
+                 System.out.println(a/i);
```
Comment (rakesh-stackmybiz) [inline Exception2.java:7]:
```suggestion
                 System.out.println(a / i);  
```
Space around operators

File: Exception2.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+public class Exception2 {
```
Comment (rakesh-stackmybiz) [inline Exception2.java:1]: Exception is a reserved class, try to avoid such class name.

File: ExceptionHandling.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
+public class ExceptionHandling {
+    int a,b,c;
```
Comment (rakesh-stackmybiz) [inline ExceptionHandling.java:2]:
```suggestion
    int a,b,c;
```
Break this down into multiple statements.

File: FinallyExeption.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+                finally {
+                    System.out.println("finally  is always occur");
+                }
+    
+                System.out.println("rest code");
+            }
+        public static void main(String args[]) {
+
+            FinallyExeption exc = new FinallyExeption();
+            exc.test();s cfdjyg fv
```
Comment (rakesh-stackmybiz) [inline FinallyExeption.java:20]:
```suggestion
            exc.test(); 
```
Syntax issue

**Comment by bikashstackmybiz** [comment]
all corection done

**Pull request merged**